src/plugins/ui_feature/autofill.ts

```diff
@@ -238,6 +247,115 @@ export class AutofillPlugin extends UIPlugin {
     }
   }
 
+  private collectBordersData(
+    data: Omit<AutoFillCellCommand, "type">,
+    borders: Record<UID, Border>,
+    bordersPositions: Record<number, Zone[]>
+  ) {
+    const id = getItemId(data.border, borders);
+    if (!(id in bordersPositions)) {
+      bordersPositions[id] = [];
+    }
+    bordersPositions[id].push(positionToZone({ col: data.col, row: data.row }));
```

Comment:
we could use unbound zones here, I don't think the borders support (or that it is possible to have) unbound zones. it would remove one property from each zone of the border: `hasHeader: false`